platform/ui/src/components/MeasurementTable/MeasurementTable.jsx

```diff
@@ -3,6 +3,70 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import { Icon } from '@ohif/ui';
 
+const MeasurementItem = ({ id, index, label, displayText, isActive, onClick, onEdit }) => {
+  const onEditHandler = event => {
+    /* Stop propagation to avoid disable the current active item */
+    event.stopPropagation();
```

Comment:
Can you provide more info? This is similar to using `!important` in CSS, and I want to make sure we don't have any other options.

Reply:
@dannyrb Edit is a button inside another, we are stopping propagation to avoid unnecessary click event that updates the active item to non-active. I don't see event handling similar to using !important yet, can you share why? I'm curious

Reply:
For example, `ViewportNotification`, when open, adds an event listener to the `document` to listen for any clicks outside of the `ViewportNotification` to "cancel"/"dismiss".

If that viewport notification is open, and you click this edit handle, it will remain open. The event never bubbles to `document`. Can we instead tell the MeasurementItem that, if it receives a click event, and it's already active, that it shouldn't toggle it's activity?